ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -390,24 +402,47 @@ export const Homepage = withUserProfile()(class extends React.Component<
                               <RecentWorkspaces />
                             </FlexColumn>
                             <FlexColumn>
-                              <SmallHeader>Recently Accessed Items</SmallHeader>
-                              <RecentResources/>
+                              {this.state.userHasWorkspaces ? (<React.Fragment>
+                                <SmallHeader>Recently Accessed Items</SmallHeader>
+                                <RecentResources/>
+                              </React.Fragment>) : <div style={{backgroundColor: colors.lightBackground,
+                                color: colors.primary,
+                                borderRadius: 10,
+                                padding: '10px 30px',
+                                margin: '30px 0px'}}>
```

Comment:
Looks like the sides of the fadebox are drawn over

Reply:
What do you mean? I don't think I'm seeing this.